### PR TITLE
Ability to specify ssh port

### PR DESCRIPTION
**Pull request by r3cha**
I’m testing it at this time, will update when all be done

### CODE REVIEW

**Comment by r3cha** [comment]
Hello I saw this https://github.com/wshihadeh/docker-deployment-action/issues/8 issue bcs have same problem
Here is simple PR
I’ve just test it it works well

**Comment by wshihadeh** [comment]
Thanks for the PR , but why the below config does not work for you ?

```
remote_docker_host: user@myswarm.com:456
```

Second , I think that you are missing some lines in your PR .ie 
https://github.com/wshihadeh/docker-deployment-action/blob/bd161443045e6fc51b34ffcb6c618b8a6b3598e2/docker-entrypoint.sh#L53

**Comment by r3cha** [comment]
@wshihadeh Thanks I see it, now I got it for me host:port it won’t works because I set `copy_stack_file: true`
so as I understand from code it will use ssh & scp that not accept host with port like `host:port`

**Comment by wshihadeh** [comment]
also https://github.com/wshihadeh/docker-deployment-action/blob/7f7346de1a3e8609a614f50e17267e6bfbcf5442/docker-entrypoint.sh#L83

**Comment by r3cha** [comment]
> also
> 
> https://github.com/wshihadeh/docker-deployment-action/blob/7f7346de1a3e8609a614f50e17267e6bfbcf5442/docker-entrypoint.sh#L83

Thanks! updated

File: docker-entrypoint.sh [hunk 1]
```diff
@@ -5,6 +5,7 @@ execute_ssh(){
   echo "Execute Over SSH: $@"
   ssh -q -t -i "$HOME/.ssh/id_rsa" \
       -o UserKnownHostsFile=/dev/null \
+      -p $INPUT_REMOTE_DOCKER_PORT \
       -o StrictHostKeyChecking=no "$INPUT_REMOTE_DOCKER_HOST" "$@"
 }
```
Comment (wshihadeh) [inline docker-entrypoint.sh:11]:
```suggestion

if [ -z "$INPUT_REMOTE_DOCKER_PORT" ]; then
  INPUT_REMOTE_DOCKER_PORT=22
fi
```

**Review by wshihadeh — CHANGES_REQUESTED** [review]

File: docker-entrypoint.sh (hunk 1 above)
Comment (r3cha) [reply docker-entrypoint.sh:11]: 22 default was stetted at action.yml but ok
```
  remote_docker_port:
     description: Remote Docker ssh port ie (22)
     required: false
     default: '22'
```

**Comment by wshihadeh** [comment]
merged and created a new release https://github.com/wshihadeh/docker-deployment-action/releases/tag/v2

**Pull request merged**